pyproject.toml

```
@@ -42,10 +42,13 @@ grpcio =  { version = "1.33.*", optional = true }
 grpcio-tools = { version = "1.33.*", optional = true }
 watchdog = { version = "*", optional = true }
 jupyterlab = { version = "^3.0", optional = true }
+ipython = {version = "^7.19.0", optional = true, python = ">=3.7,<4.0"}
```

why using an old version? if you need to use `7.19.0` the it must be
```suggestion
ipython = {version = "^7.19", optional = true, python = ">=3.7,<4.0"}
```